src/applications/vaos/components/cancel/CancelCernerAppointmentModal.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import Modal from '@department-of-veterans-affairs/formation-react/Modal';
+import { FETCH_STATUS } from '../../utils/constants';
+import { getCernerPortalLink } from '../../utils/appointment';
+
+export default function CancelCernerAppointmentModal({ onClose, status }) {
+  return (
+    <Modal
+      id="cancelCernerAppt"
+      status="warning"
+      visible
+      onClose={onClose}
+      title="You can't cancel this appointment on the VA appointments tool."
```

Comment:
Should use ’ instead of ' in can't.

Reply:
What's the reason?

Reply:
It's part of our content standards. Generally content we get from content writers will already have it in what they're providing to us, but sometimes the standard quote slips through.

Reply:
Done.